### ISSUE TITLE
get low accuracy on icdar2013 recognition test set.

### ISSUE THREAD

**Issue by zhangsiqiGit** [JaidedAI/EasyOCR#910]
Hi, it's an great project. 
But i found the default recognition english model test on ICD13 get poor acc.
{"totalWords": 1095, "detWords": 1095, "crwN": 768.0, "crwupN": 805.0, "ted": 911.0, "tedL": 197.8682213211625, "crw": 0.7013698630136986, "tedup": 739.0, "tedupL": 167.13990935535054, "crwup": 0.7351598173515982}

And i modified some code part ensure not exec detect, this result is only recognition part.
So, I'm consider what's your training data, and am I test right?
